app/services/relocation_manager.py

```
@@ -263,21 +268,33 @@ def progress_callback(bytes_transferred: int):
+                    raise Exception(msg)
+                new_file_path = storage_reference
+            else:
+                success, error = FileMover.move_file(
+                    current_file_path,
+                    new_file_path,
+                    OperationType.MOVE,
+                    progress_callback=progress_callback,
+                )
+                if not success:
+                    msg = f"File move failed: {error}"
+                    raise Exception(msg)
```

Ruff (0.15.10)

[warning] Abstract `raise` to an inner function

(TRY301)